qemu/tests/cfg/remote_negative.cfg

```diff
@@ -0,0 +1,14 @@
+# Network storage backends:
+#   iscsi_direct ceph
+# The following testing scenarios are covered:
+#   Failed to boot VM from a readonly image
+
+- remote_negative:
```

Comment:
Suggest the name `remote_storage_negative`.

Reply:
change doen